src/main.ts

```diff
@@ -440,16 +444,21 @@ export function prettifyExecutionGraphResult(
   )
 }
 
-export function displayErrorExecutionGraphFailed(executionGraph: Object): void {
+export function displayErrorExecutionGraph(
+  executionGraph: Object,
+  status: String
```

Comment:
Can we skip this parameter? It will be a bit cleaner.

Reply:
That's for sure